k9mail/src/main/java/com/fsck/k9/service/NotificationActionService.java

```diff
@@ -34,37 +35,83 @@
     private final static String EXTRA_ACCOUNT = "account";
     private final static String EXTRA_MESSAGE = "message";
     private final static String EXTRA_MESSAGE_LIST = "messages";
+    private final static String EXTRA_DONTCANCEL = "dontcancel";
```

Comment:
Yes, EXTRA_DONTCANCEL is no longer needed. I didn't notice it was still in there.